hadoop-ozone/dist/src/main/smoketest/s3/objectputget.robot

```diff
@@ -49,7 +49,7 @@ Get object from s3
 Get object with wrong signature
     Pass Execution If          '${SECURITY_ENABLED}' == 'false'    Skip in unsecure cluster
     ${result} =                 Execute and Ignore Error   curl -i -H 'Authorization: AWS scm/scm@EXAMPLE.COM:asdfqwerty' ${ENDPOINT_URL}/${BUCKET}/${PREFIX}/putobject/key=value/f1
-                                Should contain             ${result}        403 Forbidden
+                                Should contain             ${result}        400 Bad Request
```

Comment:
Looks like `403` is the expected status code for `SignatureDoesNotMatch`?

ref: https://docs.aws.amazon.com/IAM/latest/UserGuide/signature-v4-troubleshooting.html

Reply:
@smengcl Thanks for the review. 
The request here is sending an AWS v2 which we do not support and that's why I was returning a "400 - Bad request".  If a "403 - Forbidden" return code makes sense here then I can make that change.
However, for some of the other requests with invalid/incorrect AWS V4 auth headers/requests, I'm returning a "400 - Malformed auth header" exception . Will modify the error codes and error messages according to the AWS documentation.

Reply:
Thanks Saketa. `400 - Malformed auth header` sounds good. In this case the user will be informed of the correct action to take (to fix auth header).

Reply:
Kept the error message but updated the error code to 403 since that's what is expected from AWS documentation